device/android/README.md

```diff
@@ -0,0 +1,53 @@
+## Setup Android SDK
```

Comment:
Please consider adding it to
https://k2-fsa.github.io/sherpa/

Reply:
You mean adding a link or put all of them to the top README？